Rakefile

```diff
@@ -53,8 +53,17 @@ namespace :ragel do
   desc "Process the ragel source files and output ruby code"
   task :rb do |t|
     RAGEL_SOURCE_FILES.each do |file|
+      output_file = "#{RAGEL_OUTPUT_DIR}/#{file}.rb"
       # using faster flat table driven FSM, about 25% larger code, but about 30% faster
-      sh "ragel -F1 -R #{RAGEL_SOURCE_DIR}/#{file}.rl -o #{RAGEL_OUTPUT_DIR}/#{file}.rb"
+      sh "ragel -F1 -R #{RAGEL_SOURCE_DIR}/#{file}.rl -o #{output_file}"
```

Comment:
I realize this was merged, but I noticed this and `sh` can take an array and handle quoting properly, so you can write this as:

```ruby
sh %W[ragel -F1 -R #{RAGEL_SOURCE_DIR}/#{file}.rl -o #{output_file}]
```

Also, you could probably even simplify this further to:

```ruby
RAGEL_SOURCE_FILES.each do |file|
  output_file = Pathname.new("#{RAGEL_OUTPUT_DIR}/#{file}.rb")

  sh %W[ragel -F1 -R #{RAGEL_SOURCE_DIR}/#{file}.rl -o #{output_file}]

  contents = output_file.read
  output_file.write("# -*- warn-indent:false; -*-\n#{contents}")
end
```